src/System.Net.Http/src/System/Net/Http/CurlHandler/CurlHandler.SslProvider.OSX.cs

```diff
@@ -14,6 +14,8 @@ internal partial class CurlHandler : HttpMessageHandler
     {
         private static class SslProvider
         {
+            private const string RequiredOpenSSLBackendName = "SecureTransport";
```

Comment:
Similarly: Interop.Http.SecureTransportDescription.  Also, this const shouldn't really have OpenSSL in its name :smile:.

Reply:
:smile: Will fix.